components/hab/src/command/pkg/binlink.rs

```diff
@@ -79,7 +79,7 @@ pub fn binlink_all_in_pkg(
 ) -> Result<()> {
     let pkg_path = PackageInstall::load(&pkg_ident, Some(fs_root_path))?;
     for bin_path in pkg_path.paths()? {
-        for bin in fs::read_dir(fs_root_path.join(bin_path.strip_prefix("/")?))? {
```

Comment:
The `fs_root_path` here is used to find the correct symlink target, even when in a filesystem root that is not `/`.